client/src/components/Weather/index.js

```diff
@@ -0,0 +1,134 @@
+import React, { useState, useEffect } from 'react';
+import ReactDOM from 'react-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCloud } from '@fortawesome/free-solid-svg-icons';
+import { faCloudRain } from '@fortawesome/free-solid-svg-icons';
+import { faBolt } from '@fortawesome/free-solid-svg-icons';
+import { faSnowflake } from '@fortawesome/free-solid-svg-icons';
+import { faSun } from '@fortawesome/free-solid-svg-icons';
+import { faSmog } from '@fortawesome/free-solid-svg-icons';
+import './style.css';
+
+const API_KEY = "8e20c3b423d1a8139959af74dcb9cba2";
+const weatherIcon = {
+    Thunderstorm: <FontAwesomeIcon icon={faBolt} />,
+    Rain: <FontAwesomeIcon icon = {faCloudRain} />,
+    Snow:  <FontAwesomeIcon icon={faSnowflake} />,
+    Atmosphere: <FontAwesomeIcon icon={faSmog} />,
+    Clear: <FontAwesomeIcon icon={faSun} />,
+    Clouds: <FontAwesomeIcon icon={faCloud} />,
+}
+
+function Weather()  {
+    
+
+    const [icon,setIcon] = useState(undefined);
+    const [main,setMain] = useState(undefined);
+    const [celsius, setCelsius] = useState(undefined);
+    const [temp_max, setMaxTemp] = useState(null);
```

Comment:
I think it would be best to have the variable and the set variable function have the same names.  You should try and stick to one [tempMax,setTempMax] or [maxTemp,setMaxTemp]

Reply:
Thank you, I will keep that consistent.